pkg/tags/tags.go

```diff
@@ -22,11 +22,17 @@ func AddTag(key, value string, tags *common.Tags) {
 
 //GetTagValue
 func GetTagValue(key string, tags *common.Tags) (string, error) {
+	var ret string
 	for _, tag := range tags.GetTags() {
 		if tag.GetKey() == key {
-			return tag.GetValue(), nil
+			ret = tag.GetValue()
 		}
```

Comment:
Not sure if we have duplicate values.

@raghavendra-nataraj - can you take a look